debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -252,4 +279,14 @@ private static int validateCollectionBlacklist(Configuration config, Field field
         }
         return 0;
     }
+    
+    private static int validateDatabaseBlacklist(Configuration config, Field field, ValidationOutput problems) {
```

Comment:
I see your point but was trying to be consistent within the mongo connector.  The collection.whitelist and collection.blacklist are currently coded to be mutually exclusive (not sure why).  If the current collection setting were not coded this way, I would be able to combine collection.whitelist and collection.blacklist patterns that accomplish what I need without ever needing the settings specifically at the database level (possible since the collection comparisons are performed while including the database name as a prefix).